gen/bobgen-sqlite/driver/sqlite_test.go

```diff
@@ -136,7 +136,7 @@ func TestAssembleSQLite(t *testing.T) {
 
 	config := Config{
 		Config: helpers.Config{
-			Dsn: mainDB.Name() + "?_pragma=foreign_keys(1)&_pragma=busy_timeout(10000)",
+			Dsn: "file:" + mainDB.Name() + "?_pragma=foreign_keys(1)&_pragma=busy_timeout(10000)",
```

Comment:
My driver is perhaps too strict, but it follows [SQLite in requiring the `file:` scheme](https://sqlite.org/uri.html#uri_format):
> The scheme is always required. (...) The scheme of the URI must be "file:".

Also [here](https://www.sqlite.org/c3ref/open.html#:~:text=URI%20Filenames,-If%20URI%20filename):
> If [URI filename](https://www.sqlite.org/uri.html) interpretation is enabled, and the filename argument begins with "file:" (...)